auth/routes/v1.ts

```diff
@@ -0,0 +1,63 @@
+import { Request, Response, Router } from "express";
+import {
+  createUser,
+  deleteUser,
+  getUser,
+  loginUser,
+  updateUser
+} from "../controllers/userController";
+
+export default function (app: Router, passport: any) {
+  /********  Api details  **********/
+  app.get("/api", function (req: Request, res: Response) {
+    res.json({ message: "Internado", version: "v1.0.0" });
+  });
+  /********************************/
+
+  // Login user
+  app.post("/user/login", loginUser);
+  app.options("/user/login", corsAllowMethods("POST"));
+
+  // Signup user
+  app.post("/user", createUser);
```

Comment:
I think it would be clearer to name this endpoint `/user/create` instead of just `/user`

Reply:
I think it is not good practice to insert the action required in the route ( except for login). By doing so we would be disregarding the RESTFUL API protocol which is using POST (create a new resource), DELETE (delete a resource) and so on.  [More details](https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9)